### PR TITLE
Updated NISAR Tutorial

### PR DESCRIPTION
**Pull request by sumantjha**
Nisar tutorial updated with instructions to access data from JPL's website.

### CODE REVIEW

File: docs/source/tutorials/ATL03/IceSat_ATL03_Tutorials.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
  "cells": [
```
Comment (spa0002) [inline docs/source/tutorials/ATL03/IceSat_ATL03_Tutorials.ipynb:2]: <p>Recommend "the Pandas"	</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (spa0002) [inline docs/source/tutorials/ATL03/IceSat_ATL03_Tutorials.ipynb:2]: <p>Recommend the Geopandas</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

File: docs/source/tutorials/NISAR/Nisar_access_explore.ipynb [hunk 2]
```diff
@@ -0,0 +1,676 @@
+{
```
Comment (spa0002) [inline docs/source/tutorials/NISAR/Nisar_access_explore.ipynb:1]: <p>Recommend adding a space between "data access, reading," and "and visualizing amplitude".		</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (spa0002) [inline docs/source/tutorials/NISAR/Nisar_access_explore.ipynb:1]: <p>Recommend changing " is available from NISAR mission is " to " is available from the NISAR mission that is ".</p><p>Recommend splitting into two sentences -&gt; "launched in 2014. At this time" and changing "will get from NISAR satellite." to "will get from the NISAR satellite."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (spa0002) [inline docs/source/tutorials/NISAR/Nisar_access_explore.ipynb:1]: <ul><li>Recommend changing "Jupyter workspace with necessary python packages." to "Jupyter workspace with the necessary python packages."</li><li>Recommend adding a space between "os, maap, h5glance," and "osgeo, and matplotlib.&nbsp;"</li><li>Recommend changing "check and install packages required for" to " check and install the packages required for".</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (spa0002) [inline docs/source/tutorials/NISAR/Nisar_access_explore.ipynb:1]: <p>Recommend:</p><ul><li>the xcode</li><li>outside the Jupyter</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (spa0002) [inline docs/source/tutorials/NISAR/Nisar_access_explore.ipynb:1]: <p>Recommend:</p><ul><li>the above</li><li>get an H5</li><li>Users's -&gt; Users</li><li>the UAVSAR</li><li>on the map</li><li>it -&gt; them</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (spa0002) [inline docs/source/tutorials/NISAR/Nisar_access_explore.ipynb:1]: <p>Recommend:</p><ul><li>the "wget"</li><li>sometime -&gt; some time</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (spa0002) [inline docs/source/tutorials/NISAR/Nisar_access_explore.ipynb:1]: <p>Recommend "the name".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (spa0002) [inline docs/source/tutorials/NISAR/Nisar_access_explore.ipynb:1]: <p>Recommend "the product".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (spa0002) [reply docs/source/tutorials/NISAR/Nisar_access_explore.ipynb:1]: <p>the names</p>

Comment (spa0002) [reply docs/source/tutorials/NISAR/Nisar_access_explore.ipynb:1]: <p>the names</p>

**Review by emileten — APPROVED** [review]
Thanks for the updates @sumantjha, looks good to me. Optional minor suggestion : 

> The block below will check if your packages are available and install them if they are not.
> # pip install h5py numpy glob h5glance osgeo matplotlib


Might want to change the block to :

> # ! pip install h5py numpy glob h5glance osgeo matplotlib
 
(note the exclamation mark)

**Pull request merged**